.github/workflows/pr-comment.yaml

```
@@ -1,47 +1,91 @@
+            // Set outputs
+            const runUrl = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${context.payload.workflow_run.id}`;
+            const fullMessage = `${config.message}\n\nFor more info: ${runUrl}`;
+
+            core.setOutput("comment_tag", config.commentTag);
+            core.setOutput("message", fullMessage);
       - name: Unzip artifact
         run: unzip pr_number.zip
-      - name: Extract data
+      - name: Extract PR number
         id: artifact-data
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:48: Double quote to prevent globbing and word splitting

(shellcheck)